Engine/source/scene/sceneContainer.cpp

```diff
@@ -1012,6 +1012,11 @@ bool SceneContainer::_castRay( U32 type, const Point3F& start, const Point3F& en
       F32 currStartX = normalStart.x;
 
       AssertFatal(currStartX != normalEnd.x, "This is going to cause problems in SceneContainer::castRay");
+	  if(_isnan(currStartX))
```

Comment:
hmm @eightyeight is there some alternative? or could some wrapper code be added? Would love for torque to be multi-platform. For our projects we have just used windows, so I often don't take the time to make sure it's multi-platform code we write.

I suppose a platform check could be added, but this was added because of a bug we experienced.